Release 2.2.0 (#2599)
nginxinc/kubernetes-ingress: README.md

We publish Ingress controller releases on GitHub. See our [releases page](https://github.com/nginxinc/kubernetes-ingress/releases).

> The latest stable release is [2.2.0](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v2.2.0). For production use, we recommend that you choose the latest stable release.  As an alternative, you can choose the *edge* version built from the [latest commit](https://github.com/nginxinc/kubernetes-ingress/commits/main) from the main branch. The edge version is useful for experimenting with new features that are not yet published in a stable release.

Reviewer: This should be rewritten as follows and its own paragraph:

The edge version is useful for experimenting with new features that are not yet published in a stable release.  To choose the *edge* version built from the [latest commit](https://github.com/nginxinc/kubernetes-ingress/commits/main) from the main branch.

Reviewer 2: So something like this?
```suggestion
The latest stable release is [2.2.0](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v2.2.0). For production use, we recommend that you choose the latest stable release. 

The edge version is useful for experimenting with new features that are not yet published in a stable release. To choose the edge version built from the [latest commit](https://github.com/nginxinc/kubernetes-ingress/commits/main) from the main branch.
```
